prov/efa/src/efa_hmem.c

```diff
@@ -153,19 +184,38 @@ static int efa_hmem_support_status_update_neuron(struct efa_hmem_support_status
 	}
 
 	neuron_status->p2p_supported = true;
+	/* Eager and runting read protocols */
+	if (-ENODATA != fi_param_get(&rxr_prov, "inter_max_medium_message_size", &tmp_value)) {
+		EFA_WARN(FI_LOG_DOMAIN,
+		         "The  environment variable FI_EFA_INTER_MAX_MEDIUM_MESSAGE_SIZE was set, "
```

Comment:
Extra space between `The` and `environment`

Reply:
done